plugins/arDominionB5Plugin/modules/accession/templates/indexSuccess.php

```diff
@@ -155,7 +155,7 @@
 
 <div class="section" id="rightsArea">
 
-  <?php echo link_to_if(QubitAcl::check($resource, 'update'), '<h2>'.__('Rights area').'</h2>', [$resource, 'module' => 'accession', 'action' => 'edit'], ['anchor' => 'rightsArea', 'title' => __('Edit rights area')]); ?>
+  <?php echo link_to_if(QubitAcl::check($resource, 'update'), '<h2>'.__('Rights area').'</h2>', [$resource, 'module' => 'accession', 'action' => 'edit'], ['anchor' => 'io-collapse', 'title' => __('Edit rights area')]); ?>
```

Comment:
I believe this one was broken as there is no `rightsArea` in the current template either. Maybe remove this heading link and, if you want, add it to the `informationObjectArea` heading below.

Reply:
Ah I understand now. The accession I was testing with had no rights information or archival information object attached so I was not able to see those sections :+1: I think I've fixed it in https://github.com/artefactual/atom/pull/1343/commits/731774e1e1c62288aeba28794d1e918fefded26c